frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSection.tsx

```diff
@@ -48,6 +50,7 @@ export const AuthSettingsSection = ({
   settings,
 }: AuthSettingsSectionProps): JSX.Element => {
   const { mutateFormAuthType } = useMutateFormSettings()
+  const { data: form } = useAdminForm()
```

Comment:
might want to retrieve beta flags from the user object itself (from `useUser` hook over `useAdminForm`

Reply:
ooo you're right. Thanks!